packages/devtools_app/test/timeline_controller_test.dart

```diff
@@ -2,18 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-@TestOn('vm')
-import 'package:test/test.dart';
-
-import 'package:devtools_testing/timeline_controller_test.dart';
 import 'package:devtools_testing/support/flutter_test_driver.dart'
     show FlutterRunConfiguration;
 import 'package:devtools_testing/support/flutter_test_environment.dart';
+import 'package:devtools_testing/timeline_controller_test.dart';
+@TestOn('vm')
+import 'package:test/test.dart';
 
 void main() async {
-  final FlutterTestEnvironment env = FlutterTestEnvironment(
-    const FlutterRunConfiguration(withDebugger: true),
-  );
+  // TODO(devoncarew): Skip the timeline_controller_test.dart tests (#1778).
+  // ignore: dead_code
+  if (false) {
+    final FlutterTestEnvironment env = FlutterTestEnvironment(
+      const FlutterRunConfiguration(withDebugger: true),
+    );
 
-  await runTimelineControllerTests(env);
+    await runTimelineControllerTests(env);
+  }
```

Comment:
you may also need to make this change to devtools_app/test/flutter/timeline_controller_test

Reply:
Cool; I'll add that test to the exclusion if we see a flake from it. I'm hoping these tests won't have to be disabled long.